docs/SDL/OnStatusUpdate/index.md

```diff
@@ -20,9 +20,9 @@ Inform the User about the status of PTU by updating the appropriate UI screens.
 !!! note
 
 SDL operates with the following PTU statuses:
-* UPDATE_NEEDED - one of the triggers for PTU occurs (see [BC.PolicyUpdate](../../BasicCommunication/PolicyUpdate/index.md#Request))
-* UPDATING - SDL starts the PTU flow by creating the Snapshot of Local Policy Table
-* UP_TO_DATE - SDL merges the Updated Policy Table to the Local one
+* `UPDATE_NEEDED` - one of the triggers for PTU occurs (see [BC.PolicyUpdate](../../BasicCommunication/PolicyUpdate/index.md#Request))
```

Comment:
The bullet point doesn't show up correctly on the dev portal so maybe try using `-` instead of `*`
![Screen Shot 2020-01-24 at 2 18 08 PM](https://user-images.githubusercontent.com/10703941/73097201-9995fb00-3eb4-11ea-8afb-7117faa1a286.png)
```suggestion
- `UPDATE_NEEDED` - one of the triggers for PTU occurs (see [BC.PolicyUpdate](../../BasicCommunication/PolicyUpdate/index.md#Request))
```